### DISCUSSION TITLE
Issues with learn_one and the documentation

### DISCUSSION THREAD

**Discussion by hairuoguo** [online-ml/river#1470] [General]
Hello! 

I am currently trying to use this library for a project, but there are a couple of things that have been blocking my progress.

1. The documentation site is down -- would it be possible to have the documentation hosted on another site or to revive the documentation page? It would make it a lot easier to figure out what functions are supposed to do/the intended types and descriptions for certain arguments.
2. I'm currently trying to work with the SoftMax model in linear_models. However, I've been having trouble with the learn_one method for that model. To be more precise, I end up with a "unsupported operand type(s) for *: 'dict' and 'float'" error for line 96 in linear_model/softmax. My x input is a dictionary with str keys for features and float values for feature values (input dim size 1, this is a toy problem), and I've tried formatting the y both as a single int/str class label and as a dictionary of floats (keys as labels, values as desired confidence scores). What would be the appropriate formatting to ensure that the learn/predict functions work properly? (some syntactic sugar where the functions are declared would also be helpful in lieu of complete documentation). Thank you!

**Comment by MaxHalford** [comment]
Hey there.

> The documentation site is down -- would it be possible to have the documentation hosted on another site or to revive the documentation page? It would make it a lot easier to figure out what functions are supposed to do/the intended types and descriptions for certain arguments.

Which site are you talking about? https://riverml.xyz/latest/ is up for me

> I'm currently trying to work with the SoftMax model in linear_models. However, I've been having trouble with the learn_one method for that model. To be more precise, I end up with a "unsupported operand type(s) for *: 'dict' and 'float'" error for line 96 in linear_model/softmax. My x input is a dictionary with str keys for features and float values for feature values (input dim size 1, this is a toy problem), and I've tried formatting the y both as a single int/str class label and as a dictionary of floats (keys as labels, values as desired confidence scores). What would be the appropriate formatting to ensure that the learn/predict functions work properly? (some syntactic sugar where the functions are declared would also be helpful in lieu of complete documentation). Thank you!

How about sharing an [MRE](https://stackoverflow.com/help/minimal-reproducible-example) with some code?

**Comment by hairuoguo** [comment]
Ah, apparently the documentation site wasn't loading for me due to my being on a VPN, but am able to access it once VPN is turned off. Was able to resolve issue by looking at documentation. Thanks!
